dusk.md

```diff
@@ -1317,3 +1318,25 @@ To run your Dusk tests on Travis CI, we will need to use the "sudo-enabled" Ubun
 
     script:
        - php artisan dusk
+
+<a name="running-tests-on-gitlab-ci-cd"></a>
+### Gitlab 
+
+Running your Dusk tests on Gitlab's CI/CD pipelines works easiest by explicitly running the chromedriver executable as well as PHPs development server using `php artisan serve`. See the following example `.gitlab-ci.yml` file.
+
+    Tests:
+      image: chilio/laravel-dusk-ci:stable
```

Comment:
I think it should be image maintained by Laravel.

Reply:
Yes, that would be better. 

Is there an image maintained by laravel? Or should I just remove the line because it is non-laravel-related and Gitlab specific?

The set of commands is whats required for specifically dusk to work, the image is up to the developer knowing their way around Gitlab CI/CD.

Reply:
We just need a confirmation from Laravel maintainers on this. I would leave it there for now.